pkg/agent/prommetrics.go

```diff
@@ -76,6 +186,24 @@ func makePrometheusParts(globalstate *agentState) (PromMetrics, *prometheus.Regi
 		)),
 	}
 
+	// Some of of the metrics should have default keys set to zero. Otherwise, these won't be filled
+	// unil the value is non-zero (because something's happened), which makes it harder to
+	// distinguish between "valid signal of nothing" vs "no signal".
+	metricsWithDirection := []resourceChangePair{
+		// sched requested/approved
+		metrics.schedulerRequestedChange, metrics.schedulerApprovedChange,
+		// informant requested/approved
+		metrics.informantRequestedChange, metrics.informantApprovedChange,
+		// neonvm requested
+		metrics.neonvmRequestedChange,
+	}
+	for _, p := range metricsWithDirection {
+		for _, m := range []*prometheus.CounterVec{p.cpu, p.mem} {
+			m.WithLabelValues("up").Add(0.0)
+			m.WithLabelValues("down").Add(0.0)
```

Comment:
I'd probably go for inc/dec rather than up/down and just write a helper to do this inline when you declare the metrics

Reply:
Done